### PR TITLE
Prediction for bayesian models

### PR DESCRIPTION
**Pull request by ShouvikGhosh2048**

### CODE REVIEW

File: src/bayesian/getter.jl [hunk 1]
```diff
@@ -8,4 +8,40 @@ function predict(container::BayesianRegression{:LinearRegression}, newdata::Data
     end
     predictions = params.α' .+ X * W'
     return vec(mean(predictions, dims=2))
+end
+
+function predict(container::BayesianRegression{:LogisticRegression}, newdata::DataFrame, prediction_chain_start::Int64 = 200)
+    X = modelmatrix(container.formula, newdata)
+
+    params = get_params(container.chain[prediction_chain_start:end,:,:])
+    W = params.β
```
Comment (ayushpatnaikgit) [inline src/bayesian/getter.jl:17]:
```suggestion
function predict(container::BayesianRegression{:LogisticRegression}, newdata::DataFrame, prediction_chain_start::Int64 = 200)
    X = modelmatrix(container.formula, newdata)

    params = get_params(container.chain[prediction_chain_start:end,:,:]) #getting parameters
    W = params.β
```

**Comment by codetalker7** [comment]
Looks good to me. Only one question: @ShouvikGhosh2048 was your query about the intercept term resolved?

File: src/bayesian/getter.jl [hunk 2]
```diff
@@ -8,4 +8,40 @@ function predict(container::BayesianRegression{:LinearRegression}, newdata::Data
...
+    if isa(W, Tuple)
+        W = reduce(hcat, W)
+    end
+    z = X * W'
+    return vec(mean(container.link.link.(z), dims=2))
```
Comment (codetalker7) [inline src/bayesian/getter.jl:22]: Will it be possible to change the name of the link property? `contained.link.link` makes it a bit ambiguous.

**Review by codetalker7 — APPROVED** [review]

**Comment by ShouvikGhosh2048** [comment]
> Looks good to me. Only one question: @ShouvikGhosh2048 was your query about the intercept term resolved?

Not till now, I'll ask next time we meet.

**Comment by sourish-cmi** [comment]
> > Looks good to me. Only one question: @ShouvikGhosh2048 was your query about the intercept term resolved?
> 
> Not till now, I'll ask next time we meet.

If you are talking about the missing intercept terms in logistic regression, it is being solved with this [PR](https://github.com/xKDR/CRRao.jl/pull/29) and it is being merged

**Pull request merged**